impeller/entity/contents/content_context.h

```diff
@@ -992,14 +955,13 @@ class ContentContext {
   mutable Variants<VerticesUberShader> vertices_uber_shader_;
 
   template <class TypedPipeline>
-  const Pipeline<PipelineDescriptor>* GetPipeline(
-      Variants<TypedPipeline>& container,
-      ContentContextOptions opts) const {
+  PipelineRef GetPipeline(Variants<TypedPipeline>& container,
```

Comment:
We should have a comment somewhere that says why we are able to pass around raw_ptrs.  This is the best place I can think of.  Or maybe at the typedef?

Reply:
Done, on the typedef.